voltdbclient/response.go

```diff
@@ -303,7 +303,8 @@ func deserializeTableForResult(r io.Reader) (rowsAff int64, err error) {
 	if err != nil {
 		return 0, err
 	}
-	if strings.Compare("modified_tuples", cname) != 0 {
+
+	if strings.Compare("modified_tuples", cname) != 0 && cname != "STATUS" {
```

Comment:
> The response can have any columns in the table. We shouldn’t assume anything.

this is different, there re two modes of querying. Using `Exec` which will retrurn `Result` describing the numbers of rows affected by the operation and the LastInsert ID if any.

So It is correct to check `modified_tuples` since it gives us the values of the rows affected by the operation if I'm correct.

The case os `STATUS` is a special case, which we will now address in this PR

> The original code used strings.Compare(), which is less preferred. It would be nice to be consistent on using != or == if we decide to keep this check

I prefer to use `!=` I kept it to avoid drastic change, seeing you veiew this way I think we can as well go with `!=` check